config/initializers/curation_concerns_actors.rb

```diff
@@ -0,0 +1,2 @@
+CurationConcerns::Actors::ActorFactory.remove_actor CurationConcerns::Actors::AddToCollectionActor
```

Comment:
This is breaking. These ActorFactory methods aren't defined anywhere?
https://travis-ci.org/ualbertalib/CollectionNorth